content/posts/2020-09-08-analiza-wynikow/index.md

```diff
@@ -0,0 +1,121 @@
+---
+title: "Smutne wyniki testu C++"
+subtitle: "Dwa lata później"
+date: 2020-09-07T12:00:00+02:00
+lastmod: 2020-09-07T12:00:00+02:00
+draft: false
+author: ""
+authorLink: ""
+description: ""
+summary: ""
+url: "/post/analiza-wyników"
+
+tags: ["c++", "kurs", "online", "programowanie"]
+categories: ["Post", "Raport"]
+keywords: ["Kurs C++", "wyniki testu c++", "analiza wyników"]
+hiddenFromHomePage: false
+hiddenFromSearch: false
+
+featuredImage: ""
+featuredImagePreview: ""
+
+toc:
+  enable: false
+math:
+  enable: false
+lightgallery: false
+share:
+  enable: true
+comment:
+  enable: true
+license: ""
+---
+
+W latach 80-tych duńczyk Bjarne Stroustrup stworzył rozszerzenie obiektowe do języka C. Jako że była to kolejna iteracja języka, ostatecznie nazwano ją C++.
+<!-- Na chwilę obecną nie mam lepszego pomysłu co wpisać w tym tekście. Nie mam również pomysłu na featuredImage -->
+
+<!--more-->
+
+## Wstęp
+
+Obecnie C++ jest jednym z najpopularniejszych języków programowania używanym w branży IT. Jego znajomość jest czynnikiem istotnym przy szukaniu zatrudnienia jako programista.
+Misją Coders School jest kształcenie kompetencji w aspirujących programistach, aby zapewnić im wszechstronny rozwój, w stopniu lepszym, niż jest to prowadzone na uczelniach czy bootcampach. Po naszych kursach o wiele łatwiej będzie Ci wykazać się w trakcie rozmowy o pracę. Nie skupiamy się jedynie na umiejętnościach twardych, ale również przykładamy uwagę do umiejętności miękkich, które są równie ważne w IT.
+Nauka u nas odbywa się nie w sposób znany z uczelni, a na zasadzie grywalizacji. Za wykonywane zadania zdobywasz punkty doświadczenia, wbijasz kolejne poziomy, odblokowujesz osiągnięcia i otrzymujesz nagrody jak w grze. Przy okazji zdobywasz przydatną wiedzę i łączysz przyjemne z pożytecznym.
+Na naszej stronie dostępny jest test, w którym możesz sprawdzić swoją wiedzę z C++. Rozwiązać go może każdy, niezależnie od poziomu wiedzy i umiejętności.  
+
+## „A komu to potrzebne? A dlaczego?”
+
+Test składa się z ośmiu zamkniętych pytań o zróżnicowanym poziomie trudności. Pytania dotyczą wiedzy w zakresie znajomości C++. Odpowiedzi w teście można udzielać wielokrotnie, a sam test jest anonimowy. Możesz więc potraktować go jako sprawdzenie swojej wiedzy i umiejętności, które powinno Ci zająć około 10 minut.
+
+## Omówienie i porównanie wyników ogólnych
+
+W dniu sporządzania tego wpisu (22.08.2020) test rozwiązano 1110 razy. Przypuszczalnie większość osób, które podjęły się rozwiązania testu, to osoby zainteresowane programowaniem, które chcą rozpocząć jego naukę, są w jej trakcie albo pracują już w branży IT.
+Poniżej przedstawiam wyniki na dzień sporządzania wpisu:
+
+<div style="text-align:center;font-size:0.8em; margin-bottom:3em">
+  <img src="img\wyniki.png"
+       alt="wyniki testu"
+       style="display:block;margin:0 auto;"
+  />
+Wykres 1. Rozkład wyników testu z dnia 22.08.2020 (ilość odpowiedzi – 1110).
+</div>
```

Comment:
Użyj tagu `figure`
https://hugoloveit.com/theme-documentation-built-in-shortcodes/#figure

To odnośnie wszystkich obrazków.